crates/mithril-oxide-sys/cpp/IR/Operation.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <memory>
+
+#include <mlir/IR/Operation.h>
+#include <mlir/IR/OperationSupport.h>
+#include <mlir/Dialect/SCF/IR/SCF.h>
+
+#include <rust/cxx.h>
+
+
+namespace mithril_oxide_sys {
+
+using mlir::Operation;
+
+
+rust::Str Operation_getName(Operation &op);
+
+
+
+} // namespace mithril_oxide_sys
```

Comment:
A single blank line here is enough. I usually leave two blank lines to separate meaningful sections of code, like for example `using` and function declarations just above.